sphinx/about/whats-new.rst

```diff
@@ -1,170 +1,192 @@
+   - improved support to handle datasets generated by the Harmony software
 
 * TIFF
 
-  - split IFDs into separate series if the dimensions or pixel type mismatch
-  - restricted use case for legacy TIFF JAI reader
+   - split IFDs into separate series if the dimensions or pixel type mismatch
+   - restricted use case for legacy TIFF JAI reader
+   - fixed a bug with FillOrder which resulted in 0 pixel values
+
+* Zeiss CZI
+
+   - reduced duplicate original metadata when reading a pyramid file
 
 * Zeiss TIFF
 
-  - added support for AVI files acquired with Keyence software
+   - added support for AVI files acquired with Keyence software
 
 * Zeiss ZVI
 
-  - reuse stream for sequential calls to ``openBytes`` on the same plane
+   - reuse stream for sequential calls to ``openBytes`` on the same plane
 
 * updated all pyramidal format readers to consume ``SubResolutionReader``
 * updated all readers to consume ``MetadataTools`` getter to retrieve enumerations
 * reviewed all readers and plugins to close open instances of
   ``RandomAccessInputStream``
+* fixed some deprecation warnings in a number of readers
+* all channel metadata is now copied by ``ChannelSeparator`` instead of just names
 
-ImageJ plugin improvements
+ImageJ plugin improvements:
 
 * updated the updater message in the Fiji plugin (thanks to Jan Eglinger)
 * disabled LUT writing for any plane that has a default grayscale lookup table
 * added macro option to always skip LUT writing
 
-Command-line tools improvements
+MATLAB toolbox improvements:
+
+* improved performance of bfGetPlane by removing an unnecessary data copy (thanks to Cris Luengo)
+
+Command-line tools improvements:
 
 * ``bfconvert`` utility
 
-  - added ``-no-flat`` option to the command-line tools to convert files with 
-    sub-resolutions
-  - added ``-pyramid-scale`` and ``-pyramid-resolutions`` options to
-    generate sub-resolutions during conversion
-  - removed ``Plate`` elements when ``-series`` is passed as an option
-  - extended usage to describe available formats, extensions and compressions
+   - added ``-no-flat`` option to the command-line tools to convert files with 
+     sub-resolutions
+   - added ``-pyramid-scale`` and ``-pyramid-resolutions`` options to
+     generate sub-resolutions during conversion
+   - removed ``Plate`` elements when ``-series`` is passed as an option
+   - extended usage to describe available formats, extensions and compressions
 
 * ``xmlvalid`` utility
 
-  - added new ``validate`` methods to ``loci.formats.tools.XMLValidate`` returning
-    the validation status
-  - added a return code to ``xmlvalid``
+   - added new ``validate`` methods to ``loci.formats.tools.XMLValidate`` returning
+     the validation status
+   - added a return code to ``xmlvalid``
 
 
 Component changes:
 
-* `ome-common` was upgraded to 6.0.0-m2
+* `ome-common` was upgraded to 6.0.0
 * `ome-codecs` was upgraded to 0.2.3
```

Comment:
Along with the `ChannelSeparator` change, could also include the fact the `ome-model` was upgraded to 6.0.0-m1